2022-01-13 11:56:17.0896|INFO|Info|User Iserial:14
2022-01-13 11:56:17.1056|ERROR|Info|{"Detail":{"InnerException":null,"Message":"","StackTrace":"   at CCWFM.Web.Service.RouteCard.RouteCardService.UpdateRoutCard(RouteCardHeader header, List`1 details, Int32 postOrNo, String transactionGuid, Int32 userIserial) in D:\\CCWFM\\CCWFM.Web\\Service\\RouteCard\\RouteCardOperations.cs:line 396\r\n   at SyncInvokeUpdateRoutCard(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Exception"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"","Reason":{"Translations":[{"XmlLang":"en-US","Text":""}]},"Data":{},"InnerException":null,"StackTrace":"   ?? CCWFM.ViewModel.RouteCardViewModelClasses.RouteCardHeaderViewModel.<SaveRouteCard>b__214_1(Object s, UpdateRoutCardCompletedEventArgs e)\r\n   ?? CCWFM.RouteCardService.RouteCardServiceClient.OnUpdateRoutCardCompleted(Object state)"}
2022-01-13 11:57:41.6493|INFO|Info|User Iserial:14
2022-01-13 11:57:41.6493|ERROR|Info|{"Detail":{"InnerException":null,"Message":"","StackTrace":"   at CCWFM.Web.Service.RouteCard.RouteCardService.UpdateRoutCard(RouteCardHeader header, List`1 details, Int32 postOrNo, String transactionGuid, Int32 userIserial) in D:\\CCWFM\\CCWFM.Web\\Service\\RouteCard\\RouteCardOperations.cs:line 396\r\n   at SyncInvokeUpdateRoutCard(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Exception"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"","Reason":{"Translations":[{"XmlLang":"en-US","Text":""}]},"Data":{},"InnerException":null,"StackTrace":"   ?? CCWFM.ViewModel.RouteCardViewModelClasses.RouteCardHeaderViewModel.<SaveRouteCard>b__214_1(Object s, UpdateRoutCardCompletedEventArgs e)\r\n   ?? System.EventHandler`1.Invoke(Object sender, TEventArgs e)\r\n   ?? CCWFM.RouteCardService.RouteCardServiceClient.OnUpdateRoutCardCompleted(Object state)"}
2022-01-13 11:59:55.7173|INFO|Info|User Iserial:14
2022-01-13 11:59:55.7173|ERROR|Info|{"Detail":{"InnerException":null,"Message":"The result of a query cannot be enumerated more than once.","StackTrace":"   at CCWFM.Web.Service.RouteCard.RouteCardService.UpdateRoutCard(RouteCardHeader header, List`1 details, Int32 postOrNo, String transactionGuid, Int32 userIserial) in D:\\CCWFM\\CCWFM.Web\\Service\\RouteCard\\RouteCardOperations.cs:line 396\r\n   at SyncInvokeUpdateRoutCard(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Exception"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"The result of a query cannot be enumerated more than once.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"The result of a query cannot be enumerated more than once."}]},"Data":{},"InnerException":null,"StackTrace":"   ?? CCWFM.ViewModel.RouteCardViewModelClasses.RouteCardHeaderViewModel.<SaveRouteCard>b__214_1(Object s, UpdateRoutCardCompletedEventArgs e)\r\n   ?? System.EventHandler`1.Invoke(Object sender, TEventArgs e)\r\n   ?? CCWFM.RouteCardService.RouteCardServiceClient.OnUpdateRoutCardCompleted(Object state)"}
2022-01-13 14:29:06.0878|INFO|Info|User Iserial:1531
2022-01-13 14:29:06.0878|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/BankDepositOp/BankDepositService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.BankDepositServiceClientChannel.EndUpdateOrInsertCashDepositHeader(Int32& outindex, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.CCWFM.BankDepositService.BankDepositService.EndUpdateOrInsertCashDepositHeader(Int32& outindex, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.OnEndUpdateOrInsertCashDepositHeader(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.BankDepositService.UpdateOrInsertCashDepositHeaderCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.Gl.CashDepositViewModel.<.ctor>b__6_33(Object s, UpdateOrInsertCashDepositHeaderCompletedEventArgs x)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.OnUpdateOrInsertCashDepositHeaderCompleted(Object state)"}
2022-01-13 14:31:56.0590|INFO|Info|User Iserial:1531
2022-01-13 14:31:56.0590|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":{"InnerException":null,"Message":"The INSERT statement conflicted with the FOREIGN KEY constraint \"FK_TblLedgerMainDetail_TblAccount\". The conflict occurred in database \"sw\", table \"dbo.TblAccount\", column 'Iserial'.\r\nThe statement has been terminated.","StackTrace":"   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   at System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)\r\n   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   at System.Data.Mapping.Update.Internal.DynamicUpdateCommand.Execute(UpdateTranslator translator, EntityConnection connection, Dictionary`2 identifierValues, List`1 generatedValues)\r\n   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)","Type":"System.Data.SqlClient.SqlException"},"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at CCWFM.Web.Service.BankDepositOp.BankDepositService.UpdateOrInsertCashDepositHeader(TblCashDepositHeader newRow, String company, Int32 index, Int32 userIserial, Int32& outindex) in D:\\CCWFM\\CCWFM.Web\\Service\\BankDepositOp\\BankDeposit.cs:line 168\r\n   at SyncInvokeUpdateOrInsertCashDepositHeader(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Data.UpdateException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.BankDepositServiceClientChannel.EndUpdateOrInsertCashDepositHeader(Int32& outindex, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.CCWFM.BankDepositService.BankDepositService.EndUpdateOrInsertCashDepositHeader(Int32& outindex, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.OnEndUpdateOrInsertCashDepositHeader(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.BankDepositService.UpdateOrInsertCashDepositHeaderCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.Gl.CashDepositViewModel.<.ctor>b__6_33(Object s, UpdateOrInsertCashDepositHeaderCompletedEventArgs x)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.OnUpdateOrInsertCashDepositHeaderCompleted(Object state)"}
2022-01-13 14:51:38.2961|INFO|Info|User Iserial:1531
2022-01-13 14:51:38.3071|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":{"InnerException":null,"Message":"The INSERT statement conflicted with the FOREIGN KEY constraint \"FK_TblLedgerMainDetail_TblAccount\". The conflict occurred in database \"sw\", table \"dbo.TblAccount\", column 'Iserial'.\r\nThe statement has been terminated.","StackTrace":"   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   at System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)\r\n   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   at System.Data.Mapping.Update.Internal.DynamicUpdateCommand.Execute(UpdateTranslator translator, EntityConnection connection, Dictionary`2 identifierValues, List`1 generatedValues)\r\n   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)","Type":"System.Data.SqlClient.SqlException"},"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at CCWFM.Web.Service.BankDepositOp.BankDepositService.UpdateOrInsertCashDepositHeader(TblCashDepositHeader newRow, String company, Int32 index, Int32 userIserial, Int32& outindex) in D:\\CCWFM\\CCWFM.Web\\Service\\BankDepositOp\\BankDeposit.cs:line 168\r\n   at SyncInvokeUpdateOrInsertCashDepositHeader(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Data.UpdateException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.BankDepositServiceClientChannel.EndUpdateOrInsertCashDepositHeader(Int32& outindex, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.CCWFM.BankDepositService.BankDepositService.EndUpdateOrInsertCashDepositHeader(Int32& outindex, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.OnEndUpdateOrInsertCashDepositHeader(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.BankDepositService.UpdateOrInsertCashDepositHeaderCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.Gl.CashDepositViewModel.<.ctor>b__6_33(Object s, UpdateOrInsertCashDepositHeaderCompletedEventArgs x)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.OnUpdateOrInsertCashDepositHeaderCompleted(Object state)"}
2022-01-13 15:12:19.0632|INFO|Info|User Iserial:1531
2022-01-13 15:12:19.0732|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":{"InnerException":null,"Message":"The INSERT statement conflicted with the FOREIGN KEY constraint \"FK_TblLedgerMainDetail_TblAccount\". The conflict occurred in database \"sw\", table \"dbo.TblAccount\", column 'Iserial'.\r\nThe statement has been terminated.","StackTrace":"   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   at System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)\r\n   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   at System.Data.Mapping.Update.Internal.DynamicUpdateCommand.Execute(UpdateTranslator translator, EntityConnection connection, Dictionary`2 identifierValues, List`1 generatedValues)\r\n   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)","Type":"System.Data.SqlClient.SqlException"},"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at CCWFM.Web.Service.BankDepositOp.BankDepositService.UpdateOrInsertCashDepositHeader(TblCashDepositHeader newRow, String company, Int32 index, Int32 userIserial, Int32& outindex) in D:\\CCWFM\\CCWFM.Web\\Service\\BankDepositOp\\BankDeposit.cs:line 168\r\n   at SyncInvokeUpdateOrInsertCashDepositHeader(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Data.UpdateException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.BankDepositServiceClientChannel.EndUpdateOrInsertCashDepositHeader(Int32& outindex, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.CCWFM.BankDepositService.BankDepositService.EndUpdateOrInsertCashDepositHeader(Int32& outindex, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.OnEndUpdateOrInsertCashDepositHeader(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.BankDepositService.UpdateOrInsertCashDepositHeaderCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.Gl.CashDepositViewModel.<.ctor>b__6_33(Object s, UpdateOrInsertCashDepositHeaderCompletedEventArgs x)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.OnUpdateOrInsertCashDepositHeaderCompleted(Object state)"}
